modules/tutorials/pages/java-tutorial/retrieving-documents.adoc

```diff
@@ -0,0 +1,130 @@
+:page-pagination:
+
+// suppress inspection "SqlResolve" for whole file
+// suppress inspection "SqlNoDataSourceInspection" for whole file
+= Retrieving documents
+
+:description: pass:q[In this section, you're going to look at two methods of retrieving documents from a collection: you will use the administration console to build simple queries, and also write a short program to retrieve documents matching certain criteria. Both of the methods will introduce `N1QL` (pronounced _nickel_), Couchbase's SQL-based query language.]
+
+[abstract]
+{description}
```

Comment:
No markup in the description and description should be short — 150 characters or less, I think

```suggestion
:description: In this section, you're going to look at two methods of retrieving documents from a collection.

[abstract]
{description}
You will use the administration console to build simple queries, and also write a short program to retrieve documents matching certain criteria.
Both of the methods will introduce N1QL (pronounced "nickel"), Couchbase's SQL-based query language.
```